staging/src/k8s.io/apiserver/pkg/storage/cacher.go

```diff
@@ -649,6 +649,9 @@ func (c *Cacher) isStopped() bool {
 }
 
 func (c *Cacher) Stop() {
+	if c.isStopped() {
```

Comment:
Note that if i don't check this the `TestCRDDeletionCascading` fails

Reply:
We share storage between main resource and subresource. I guess we can get multiple calls here for that reason.

Reply:
Done